omegatrade-app/frontend/src/app/components/company/manage-company/manage-company.component.html

```diff
@@ -0,0 +1,55 @@
+<app-header></app-header>
+<app-sidebar></app-sidebar>
+<div class="spinner" *ngIf="this.loader">
```

Comment:
`*ngIf="this.loader"`
this.loader will not work, **this** word should not be used anywhere in the html file

Reply:
this issue has been resolved.